packages/react-components/lib/themes/global-theme.tsx

```diff
@@ -2,22 +2,24 @@ import { withStyles } from '@material-ui/core/styles';
 
 export const GlobalCss = withStyles((theme) => ({
   '@global': {
+    // override accordian background to distinguish it from onipanel
     '.MuiAccordion-root': {
       backgroundColor: theme.palette.background.default,
     },
+    // override text color of disabled button to make it visible
     '.MuiButtonBase-root.Mui-disabled': {
       color: theme.palette.text.primary,
     },
-    '.MuiAppBar-root': {
-      backgroundColor: theme.palette.primary.main,
-    },
+    // highlight table header in panels
     '.MuiTableCell-head': {
       background: 'rgba(0, 0, 0, 0.1)',
       borderBottom: 'none',
     },
+    // override text color of forms to be visible in theme background colors
```

Comment:
This seems more like a problem with the theme. Is it possible to fix the theme so it has better colors instead?